backend/src/test/java/de/neuefische/henny/reisekasse/Service/UserServiceTest.java

```diff
@@ -15,7 +15,7 @@
 
 
     @Test
-    void findUserByIdTestWithExistingId(){
+    void findUserByIdTestWithExistingUserIdIdShouldGiveBackOptionOfUser(){
         // Given
         String userId = "Sven";
```

Comment:
Ich würde dann auch in den Zeilen 21, 23 die userId verwenden, statt der String-Literale